### PR TITLE
feat: prepare german localization [AR-1980]

### PR DESCRIPTION
**Pull request by gongracr**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1980" title="AR-1980" target="_blank">StoryAR-1980</a>  Prepare German Localization
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

After Astrid reviewed the string copies in our AR project, she identified a number of strings whose keys and values were either misleading or not accurate enough for users with impairments to understand the flow of the app. This PR tries to ease that and make those modifications

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-788/2/) **failed**.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-788/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3945070)(3945070)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3945070)(3945070)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/calling/controlButtons/CameraButton.kt [hunk 1]
```diff
@@ -58,7 +57,10 @@ fun CameraButton(
                     R.drawable.ic_camera_off
                 }
             ),
-            contentDescription = stringResource(id = R.string.calling_turn_camera_on_off),
+            contentDescription = stringResource(
+                id = if (isCameraOn) R.string.content_description_calling_turn_camera_off
+                else R.string.content_description_calling_turn_camera_on
+            ),
```
Comment (borichellow) [inline app/src/main/kotlin/com/wire/android/ui/calling/controlButtons/CameraButton.kt:63]: what if 
```
val (painterId, contentDescriptionId ) = 
if (isCameraOn) R.drawable.ic_camera_on to R.string.content_description_calling_turn_camera_off
else R.drawable.ic_camera_off to R.string.content_description_calling_turn_camera_on
Image(
    painter = painterResource(painterId),
    contentDescription = stringResource(contentDescriptionId),
)
```

File: app/src/main/kotlin/com/wire/android/ui/calling/controlButtons/CameraFlipButton.kt [hunk 2]
```diff
@@ -36,7 +36,10 @@ fun CameraFlipButton(
                     R.drawable.ic_flip_camera_on
                 else R.drawable.ic_flip_camera_off
             ),
-            contentDescription = stringResource(id = R.string.calling_flip_camera_on_off),
+            contentDescription = stringResource(
+                id = if (isCameraFlipped) R.string.content_description_calling_flip_camera_on
+                else R.string.content_description_calling_flip_camera_off
+            ),
```
Comment (borichellow) [inline app/src/main/kotlin/com/wire/android/ui/calling/controlButtons/CameraFlipButton.kt:42]: the same here

File: app/src/main/kotlin/com/wire/android/ui/calling/controlButtons/MicrophoneButton.kt [hunk 3]
```diff
@@ -32,7 +32,10 @@ fun MicrophoneButton(
                     R.drawable.ic_unmuted
                 }
             ),
-            contentDescription = stringResource(id = R.string.calling_hang_up_call)
+            contentDescription = stringResource(
+                id = if (isMuted) R.string.content_description_calling_unmute_call
+                else R.string.content_description_calling_mute_call
+            )
```
Comment (borichellow) [inline app/src/main/kotlin/com/wire/android/ui/calling/controlButtons/MicrophoneButton.kt:38]: and here

File: app/src/main/kotlin/com/wire/android/ui/calling/controlButtons/SpeakerButton.kt [hunk 4]
```diff
@@ -30,7 +30,10 @@ fun SpeakerButton(
                     R.drawable.ic_speaker_on
                 else R.drawable.ic_speaker_off
             ),
-            contentDescription = stringResource(id = R.string.calling_turn_speaker_on_off)
+            contentDescription = stringResource(
+                id = if (isSpeakerOn) R.string.content_description_calling_turn_speaker_off
+                else R.string.content_description_calling_turn_speaker_on
+            )
```
Comment (borichellow) [inline app/src/main/kotlin/com/wire/android/ui/calling/controlButtons/SpeakerButton.kt:36]: and here :)

**Review by MohammedMokresh — APPROVED** [review]
lgtm

**Review by borichellow — APPROVED** [review]
Nice one! 👍  just a few code-style suggestions that are not mandatory :)

**Review by yamilmedina — APPROVED** [review]
🚀

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-788/6/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3978203)(3978203)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3978203)(3978203)-dev-release.apk)

**Pull request merged**